fp/_baseConvert.js

```diff
@@ -279,6 +279,9 @@ function baseConvert(util, name, func, options) {
       setPlaceholder = true;
       func.placeholder = result.placeholder = placeholder;
     }
+    result.convert = function(options) {
+      return baseConvert(util, name, func, options);
```

Comment:
I actually tried that and it didn't work. While skimming the file didn't see any code mutating the `config` after it's being created.